### ISSUE TITLE
Anonymisation URL customizable

### ISSUE THREAD

**Issue by VerioPL** [HDInnovations/UNIT3D#449]
### Issue Type
- [ ] Bug Report
- [X] Feature Idea
- [ ] Enhancement
- [ ] Technical Discussion
- [ ] Question

### End User Info
-- UNIT3D Version: #.#.#

-- Laravel Version: #.#.#

-- PHP Version:

-- Database Driver & Version:

-- Web Server Driver & Version:

-- OS Driver and Version:

### Expected Behavior:
Customizable url variable (in config/other.php) or something like that

### Actual Behavior:
Static anon.to URL in views

### Steps To Reproduce The Problem:
1.
2.
3.

**Comment by VerioPL** [comment]
@HDVinnie If accepted please assign to me

**Comment by HVRVKVT** [comment]
anon.to  has disabled automatic redirect. May i suggest https://anonym.to/ instead? They have a simple script to anonymize all the external links.

**Comment by VerioPL** [comment]
Right, secure header make referrers anonymous. So we can remove all redirects via external anonymisation providers.
